### ISSUE TITLE
Clarify mobile Safari support

### ISSUE THREAD

**Issue by HA-LO** [jquery/jquery.com#136]
Now that jQuery 3 is out, I looked at the [browser support page](http://jquery.com/browser-support/) and saw that only the _Current_ and _Current - 1_ versions of Safari are supported (currently those should be 9.1.1 and 9.1 respectively).
Anyway, the Mobile section also mentions support for iOS 7+. If I'm not mistaken, the first iOS 7 release came with Safari 6.1 preinstalled in late 2013.
Does this mean that mobile Safari 6.1+ is actively supported by jQuery? Or does it mean that iOS 7 is only supported when mobile Safari or any other mobile browser is updated to the almost latest version?
I'm sorry if this sounds like a silly feature request. I really wished those facts were better addressed by the documentation when reporting browsers versions and OS versions in the same list.

**Comment by mgol** [comment]
> Anyway, the Mobile section also mentions support for iOS 7+. If I'm not mistaken, the first iOS 7 release came with Safari 6.1 preinstalled in late 2013.

Do you have source for that info?

I've just checked in BrowserStack and that's the user agent of iOS 7.0.4 on iPhone 5s:
```
Mozilla/5.0 (iPhone; CPU iPhone OS 7_0_4 like Mac OS X) AppleWebKit/537.51.1 (KHTML, like Gecko) Version/7.0 Mobile/11B554a Safari/9537.53
```
The `Version/7.0` part should represent the Safari version. On iOS Safari is only updated with system updates so specifying which iOS versions are supported should be enough.

In any case, we only support what we can test and we now can't e.g. test against iOS 9.0-9.2 but only 9.3 as that's what BrowserStack provides.

> saw that only the _Current_ and _Current - 1_ versions of Safari are supported (currently those should be 9.1.1 and 9.1 respectively).

"Current - 1" applies to the major number, not minor; as specified in the example:
> (Current - 1) and Current denotes that we support the current stable version of the browser and the version that preceded it. For example, if the current version of a browser is 24.x, we support the 24.x and 23.x versions.

This means we support latest Safari 9.x & 8.x.

If you think there's a better way to phrase it, we're open to proposals!

**Comment by HA-LO** [comment]
Thanks so much for the reply @mgol.

I could be wrong that iOS 7 first shipped with Safari 6.1, but I'm sure it was not Safari 7.0.4.
According to [this page](https://en.wikipedia.org/wiki/Safari_version_history) Safari 7.0.4 was only released on May 2014, while iOS 7 was introduced almost at the same time with the iPhone 5s in September/October 2013.

It's true that Safari on iOS is only updated with a system update, but most system updates don't change the OS version number but just the build number.
This means that there exist actually several versions of iOS 7 with different build numbers and different versions of Safari installed.
So this emulator that Browser Stack offers must have installed an updated version of iOS 7 with a more recent version of Safari.

Good to know that Safari 8 is still supported, I was missing the bit with the major version number. Thanks for pointing out that.

As for the phrasing, well, I was just unclear about the _Mobile_ section in the browser support page. Personally, a list including browser names along with the OS name would have saved me from asking one question. Something like "Stock browser on Android 4+" and "Safari 7+ on iOS 7+" (or just "Safari 7+ on iOS" - not sure if Safari 7 can possibly run on iOS 6). But maybe I'm the only one who's concerned about these versioning details here. If so, I apologize if my issue sounds trivial.

**Comment by mgol** [comment]
> I could be wrong that iOS 7 first shipped with Safari 6.1, but I'm sure it was not Safari 7.0.4.
> According to this page Safari 7.0.4 was only released on May 2014, while iOS 7 was introduced almost at the same time with the iPhone 5s in September/October 2013.

Desktop Safari versions don't correspond with the mobile one, especially patch versions; Mobile Safari is versioned differently. I meant that iOS 7.0.4 identified its Safari as 7.0, not the other way round. For example, user agent of Safari on iOS 9.3.2 contains the string "Version/9.0". That's all, there is no separate Safari version, it's been "Version/9.0" since iOS 9.0. That's why we write "iOS" on the browser support page, there just doesn't seem to be such a thing as a concrete Mobile Safari version.

That's different to what we have on the desktop.

> It's true that Safari on iOS is only updated with a system update, but most system updates don't change the OS version number but just the build number.

That's, again, not true on iOS. For example. the user agent of Safari on iOS 9.3.2 contains the string "iPhone OS 9_3_2".

On the desktop Safari is, indeed, versioned separately and a system update may not always bump its version. iOS is different.

> As for the phrasing, well, I was just unclear about the Mobile section in the browser support page. Personally, a list including browser names along with the OS name would have saved me from asking one question. Something like "Stock browser on Android 4+" and "Safari 7+ on iOS 7+" (or just "Safari 7+ on iOS" - not sure if Safari 7 can possibly run on iOS 6).

Maybe sth like that wouldn't be a bad idea' previously we had a horizontal table with not much space for a lot of text but that's different now.

I'd write "Safari on iOS 7+", though, not "Safari 7+ on iOS 7+" because of all that I wrote above.

**Comment by mgol** [comment]
> But maybe I'm the only one who's concerned about these versioning details here. If so, I apologize if my issue sounds trivial.

Don't apologize, if docs are unclear somewhere for some people this creates an opportunity to improve them for everyone else! :)

**Comment by HA-LO** [comment]
Thanks for the reply and the clarifications. If my post will to help to improve the docs, I'm happy that I reported it here.

**Comment by mgol** [comment]
Would you like to submit a pull request with the changes? The file to modify is [pages/browser-support.html](https://github.com/jquery/jquery.com/blob/master/pages/browser-support.html).

**Comment by mgol** [comment]
Fixed via #141.
